src/app/zap-templates/common/TestHelper.js

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
could we find a different name than 'Helper' here or is that convention?

I find names of helper/util/support/common end up being vague. ClusterTestGeneration.js?